common/mouse.go

```diff
@@ -153,53 +153,53 @@ func (m *Mouse) up(x float64, y float64, opts *MouseDownUpOptions) error {
 func (m *Mouse) Click(x float64, y float64, opts goja.Value) {
 	mouseOpts := NewMouseClickOptions()
 	if err := mouseOpts.Parse(m.ctx, opts); err != nil {
-		k6ext.Panic(m.ctx, "failed parsing options: %w", err)
+		k6ext.Panic(m.ctx, "parsing click options: %w", err)
 	}
 	if err := m.click(x, y, mouseOpts); err != nil {
-		k6ext.Panic(m.ctx, "unable to mouse click: %w", err)
+		k6ext.Panic(m.ctx, "mouse click: %w", err)
 	}
 }
 
 func (m *Mouse) DblClick(x float64, y float64, opts goja.Value) {
 	mouseOpts := NewMouseDblClickOptions()
 	if err := mouseOpts.Parse(m.ctx, opts); err != nil {
-		k6ext.Panic(m.ctx, "failed parsing options: %w", err)
+		k6ext.Panic(m.ctx, "parsing dblclick options: %w", err)
 	}
 	if err := m.dblClick(x, y, mouseOpts); err != nil {
-		k6ext.Panic(m.ctx, "unable to mouse double click: %w", err)
+		k6ext.Panic(m.ctx, "mouse double click: %w", err)
 	}
 }
 
 // Down will trigger a MouseDown event in the browser.
 func (m *Mouse) Down(x float64, y float64, opts goja.Value) {
 	mouseOpts := NewMouseDownUpOptions()
 	if err := mouseOpts.Parse(m.ctx, opts); err != nil {
-		k6ext.Panic(m.ctx, "failed parsing options: %w", err)
+		k6ext.Panic(m.ctx, "parsing down options: %w", err)
 	}
 	if err := m.down(x, y, mouseOpts); err != nil {
-		k6ext.Panic(m.ctx, "unable to mouse down: %w", err)
+		k6ext.Panic(m.ctx, "mouse down: %w", err)
 	}
 }
 
 // Move will trigger a MouseMoved event in the browser.
 func (m *Mouse) Move(x float64, y float64, opts goja.Value) {
 	mouseOpts := NewMouseDownUpOptions()
 	if err := mouseOpts.Parse(m.ctx, opts); err != nil {
-		k6ext.Panic(m.ctx, "failed parsing options: %w", err)
+		k6ext.Panic(m.ctx, "parsing move options: %w", err)
 	}
 	if err := m.down(x, y, mouseOpts); err != nil {
-		k6ext.Panic(m.ctx, "unable to move mouse: %w", err)
+		k6ext.Panic(m.ctx, "mouse move: %w", err)
 	}
 }
 
 // Up will trigger a MouseUp event in the browser.
 func (m *Mouse) Up(x float64, y float64, opts goja.Value) {
 	mouseOpts := NewMouseDownUpOptions()
 	if err := mouseOpts.Parse(m.ctx, opts); err != nil {
-		k6ext.Panic(m.ctx, "failed parsing options: %w", err)
+		k6ext.Panic(m.ctx, "parsing up options: %w", err)
 	}
 	if err := m.up(x, y, mouseOpts); err != nil {
-		k6ext.Panic(m.ctx, "unable to mouse up: %w", err)
+		k6ext.Panic(m.ctx, "mouse up: %w", err)
```

Comment:
```suggestion
		k6ext.Panic(m.ctx, "releasing mouse button: %w", err)
```

Reply:
I'm on the fence about these `mouse` ones... "pressing" and "releasing" requires a mental mapping to `down`/`up`, whereas mentioning the method is immediately clear.

So I don't think we gain much from translating these into prose and would rather go sligthly against the convention and use `mouse <method>` in these errors. But we can discuss it in that new issue.